packages/core/src/common/util/converters/converters.ts

```diff
@@ -45,26 +45,31 @@ export const calculateSpace = (experiment: ExperimentType): SpaceType => {
  * @returns
  */
 export const calculateData = (
-  _categoricalValues: CategoricalVariableType[],
-  numericValues: ValueVariableType[],
-  scoreValues: ScoreVariableType[],
+  categoricalVariables: CategoricalVariableType[],
+  valueVariables: ValueVariableType[],
+  scoreVariables: ScoreVariableType[],
   dataPoints: DataEntry[]
 ): ExperimentData[] => {
-  const scoreNames = scoreValues.map(it => it.name)
-  const enabledScoreNames = scoreValues
+  const enabledVariableNames = valueVariables
+    .filter(v => v.enabled)
+    .map(v => v.name)
+    .concat(categoricalVariables.filter(v => v.enabled).map(v => v.name))
+
+  const scoreNames = scoreVariables.map(it => it.name)
+  const enabledScoreNames = scoreVariables
     .filter(it => it.enabled)
     .map(it => it.name)
+
   return dataPoints
     .filter(dp => dp.meta.enabled && dp.meta.valid)
     .map(dp => dp.data)
     .map(
       (run): ExperimentData => ({
         xi: run
           .filter(it => !scoreNames.includes(it.name))
```

Comment:
This filter line and the `scoreNames` variable can be omitted now that the next line ensures that only variable names are used